engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-info.html

```diff
@@ -36,14 +36,29 @@ <h4 translate>Basic Information</h4>
     <h4 translate>Sync Settings</h4>
     <dl class="dl-horizontal dl-horizontal-left">
       <span ng-show="repository.content_type === 'yum'" >
-        <dt translate>Restrict to architecture</dt>
+        <dt>
+          <span translate>Restrict to architecture</span>
+          <i class="pficon-info" title="{{ 'The repository will be enabled by default on content hosts with the selected architecture. Select \'Default\' to enable regardless of architecture.' | translate }}"></i>
+        </dt>
         <dd bst-edit-select="repository.arch==='noarch'?'Default':repository.arch"
             selector="repository.arch"
             options="architectures()"
             on-save="save(repository)">
         </dd>
       </span>
 
+      <span ng-show="repository.content_type === 'yum'" >
+        <dt>
+          <span translate>Restrict to <br />OS version</span>
+          <i class="pficon-info" title="{{ 'The repository will be enabled by default on content hosts with the selected OS version.  Select \'Default\' to enable regardless of OS version.' | translate }}"></i>
+        </dt>
+        <dd bst-edit-select="repository.os_versions.length ? formatOSVersions() : 'Default'"
```

Comment:
Is it feasible to get the UI to continue showing the full OS name rather than the tag value after I made a selection?

Reply:
Sure it's feasible.   Showing the tag value was on purpose though.  It's more compact on screen and matches the value you'd have to send if you were using hammer or the API.  Saving it with 'Red Hat Enterprise Linux 7' and then seeing it saved as 'rhel-7' is a good way for me as a user to grok that that's the value that is actually sent, which I imagine is useful for users of the CLI/API.  :man_shrugging:

Reply:
I guess I can roll with that!